### PR TITLE
Re-organize CI tests

### PR DESCRIPTION
**Pull request by merrymercy**

### CODE REVIEW

**Review by zhyncs — APPROVED** [review]

File: python/sglang/srt/mem_cache/chunk_cache.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ def cache_unfinished_req(self, req: Req, token_ids: Optional[List[int]] = None):
         req.last_node = entry
 
     def insert(self):
-        raise NotImplementedError
+        raise NotImplementedError()
```
Comment (zhyncs) [inline python/sglang/srt/mem_cache/chunk_cache.py:71]: What's the reason for the changes here? https://github.com/sgl-project/sglang/blob/0c1c72a0b409f255a1fcea666705af8140da5f1e/python/sglang/srt/constrained/base_tool_cache.py#L57

Comment (merrymercy) [reply python/sglang/srt/mem_cache/chunk_cache.py:71]: I think it is recommended to raise `NotImplementedError()` instead of `NotImplementedError`

Comment (zhyncs) [reply python/sglang/srt/mem_cache/chunk_cache.py:71]: ok May we also change https://github.com/sgl-project/sglang/blob/0c1c72a0b409f255a1fcea666705af8140da5f1e/python/sglang/srt/constrained/base_tool_cache.py#L57
